crates/driver/src/infra/blockchain/mod.rs

```diff
@@ -131,10 +131,16 @@ impl Ethereum {
         &self.inner.current_block
     }
 
+    fn max_block_size(&self) -> eth::U256 {
+        match self.inner.chain.0 {
+            42161 => 500_000_000,
```

Comment:
what about making this as a constant with a name and matching against it? so we know what is the number about

Reply:
Makes sense, added